ssg/_oval.py

```diff
@@ -133,15 +132,30 @@ def read_ovaldefgroup_file(testfile):
     return body
 
 
+def get_openscap_supported_oval_version():
+    try:
+        from openscap import oscap_get_version
+        if str(oscap_get_version()).split(".") >= ["1", "2", "0"]:
```

Comment:
@mpreisler This doesn't quite do what we want it to do, but it is a nice hack otherwise:

```python
>>> '1.2.16'.split(".") >= ['1', '2', '1']
True
>>> '1.2.16'.split(".") >= ['1', '2', '2']
False
```

I'm almost tempted to leave it, but add a comment about how this doesn't compare versions how we'd like it.

Reply:
The builtin (but undocumented) `distutils.version.LooseVersion` should behave correctly and should be easy to use.

Reply:
lol, what a fail... I will fix this.

Reply:
fixed

Reply:
> The builtin (but undocumented) distutils.version.LooseVersion should behave correctly and should be easy to use.

I went with list comprehension because python docs says that you don't want to use distutils directly. I suppose it doesn't make much difference either way.